metricbeat/module/http/_meta/fields.yml

```diff
@@ -45,7 +41,3 @@
               example: Not found
               description: >
                 The HTTP status phrase
-            - name: body
```

Comment:
@adriansr I just realised that body in ECS was at the moment text so this would be a breaking change. I'm now updating ECS so it's a keyword as before and we need to update the ECS file afterwards.